community/kernel/src/main/java/org/neo4j/kernel/impl/store/format/Capability.java

```diff
@@ -57,6 +57,11 @@
      */
     POINT_PROPERTIES( CapabilityType.STORE ),
 
+    /**
+     * Temporal types are an addition to the format, not a change
+     */
+    TEMPORAL_PROPERTIES( CapabilityType.STORE ),
```

Comment:
Since this is used in combination with `POINT_PROPERTIES` in the property store, perhaps it would be enough with one capability for the both, named something like `POINT_AND_TEMPORAL_PROPERTIES`?

Reply:
I found it nice to have them split, since then the error message will exactly tell you what Capability the store is missing and might be more helpful if it distinguishes between the two.

Reply:
OK if there is a use case for having them split, then it's alright. I just thought there wasn't.